app/views/companies/show.html.erb

```diff
@@ -24,9 +24,17 @@
                 </div>
               <% end %>
             <% unless @company.user == current_user %>
-                <a class="btn btn-flat message px-0" href="#" style="width: 201px;"><i class="far fa-heart fa-lg"></i> Add to favorites</a>
-                <a class="btn btn-flat message px-0" href="#" style="width: 201px;"><i class="fab fa-telegram-plane fa-lg"></i> Send message</a>
-            <% end %>
+              <% if current_user.developer && current_user.developer.favourites.include?(@company) %>
+                <%= link_to company_favourite_path(current_user.developer.company_favourites.find_by(company: @company)), method: :delete, class: 'btn btn-flat favorite' do %>
+                  <i class="fas fa-ban fa-lg"></i> Remove from favorites
```

Comment:
When you click add to favourites, this "remove from favourites" badge expands over two lines, looks a little screwy.